Source/ZoomNet/Resources/Webinars.cs

```diff
@@ -120,6 +120,53 @@ public Task<ScheduledWebinar> CreateScheduledWebinarAsync(string userId, string
 				.AsObject<ScheduledWebinar>();
 		}
 
+		/// <summary>
+		/// Updates an existing scheduled webinar.
+		/// </summary>
+		/// <param name="webinarId">The webinar ID.</param>
+		/// <param name="topic">Webinar topic.</param>
+		/// <param name="agenda">Webinar description.</param>
+		/// <param name="start">Webinar start time.</param>
+		/// <param name="duration">Webinar duration (minutes).</param>
+		/// <param name="password">Password to join the webinar. By default the password may only contain the following characters: [a-z A-Z 0-9 @ - _ *]. Max of 10 characters. This can be updated within Zoom account settings.</param>
+		/// <param name="settings">Webinar settings.</param>
+		/// <param name="trackingFields">Tracking fields.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// The async task.
+		/// Zoom will return an HTTP 200 response when there is no webinar subscription present.
+		/// A successful update is reported by an HTTP 204 value.
+		/// </returns>
+		public Task UpdateScheduledWebinarAsync(long webinarId, string topic, string agenda, DateTime start, int duration, string password = null, WebinarSettings settings = null, IDictionary<string, string> trackingFields = null, CancellationToken cancellationToken = default)
+		{
+			var data = new JObject()
+			{
+				{ "type", 5 }
+			};
+			data.AddPropertyIfValue("topic", topic);
+			data.AddPropertyIfValue("agenda", agenda);
+			data.AddPropertyIfValue("password", password);
+			data.AddPropertyIfValue("start_time", start.ToUniversalTime().ToString("yyyy-MM-dd'T'HH:mm:ss'Z'"));
+			data.AddPropertyIfValue("duration", duration);
+			data.AddPropertyIfValue("timezone", "UTC");
+			data.AddPropertyIfValue("settings", settings);
+			data.AddPropertyIfValue("tracking_fields", trackingFields?.Select(tf => new JObject() { { "field", tf.Key }, { "value", tf.Value } }));
+
+			var task = _client
+				.PatchAsync($"webinars/{webinarId}")
+				.WithJsonBody(data)
+				.WithCancellationToken(cancellationToken)
+				.AsMessage();
+
+			if (task.Result.StatusCode == System.Net.HttpStatusCode.OK)
```

Comment:
Generally speaking, I avoid `.Result` as much as I can because it blocks while waiting. It's preferable to `await` in order to "yield" the current thread while waiting for the task to complete.

- [ ] The UpdateScheduledWebinarAsync method must be marked as `async`. In other words: `public async Task UpdateScheduledWebinarAsync`
- [ ] Change `var task = _client` to `var result = await _client`
- [ ] Add `.ConfigureAwait(false);` after `.AsMessage()`
- [ ] Replace `if (task.Result.StatusCode ==` with `if (result.StatusCode ==`

Reply:
Ironically this is what I did initially before overthinking it :)